app/views/paginable/templates/_funders.html.erb

```diff
@@ -66,16 +66,16 @@
               </button>
               <ul class="dropdown-menu">
                 <% if !customization.present? %>
-                  <li><%= link_to _('Customise'), customize_org_admin_template_path(template) %></li>
+                  <li><%= link_to _('Customise'), customize_org_admin_template_path(template), 'data-method': 'post' %></li>
```

Comment:
I like the use of AJAX here but I don't see the handlers for when the request succeeds or fails, this will leave the page unchanged unless you refresh the browser